corpdir-srv-sync/src/main/java/com/github/bordertech/corpdir/sync/common/AbstractSynchronisation.java

```diff
@@ -0,0 +1,76 @@
+package com.github.bordertech.corpdir.sync.common;
+
+import com.github.bordertech.corpdir.api.common.ApiKeyIdObject;
+import com.github.bordertech.corpdir.api.exception.NotFoundException;
+import com.github.bordertech.corpdir.api.response.DataResponse;
+import com.github.bordertech.corpdir.api.service.BasicKeyIdReadOnlyService;
+import com.github.bordertech.corpdir.api.service.BasicKeyIdService;
+import java.util.List;
+import org.apache.commons.lang.reflect.FieldUtils;
+
+/**
+ * One-way synchronisation from Source to Destination
+ *
+ * @author exiqaj

```

Comment:
Use your github name as the author.

Reply:
Haha I have done coding internally, created a patch file and applied it. Thus the username, I have fixed them now